pkgs/by-name/ki/kissfft/package.nix

```diff
@@ -2,87 +2,90 @@
   lib,
   stdenv,
   fetchFromGitHub,
+  cmake,
+  ninja,
+  pkg-config,
   fftw,
   fftwFloat,
   python3,
   datatype ? "double",
-  withTools ? false,
   libpng,
   enableStatic ? stdenv.hostPlatform.isStatic,
   enableOpenmp ? false,
   llvmPackages,
+
+  # Deprecated in 24.11.
+  withTools ? null,
 }:
-let
-  py = python3.withPackages (ps: with ps; [ numpy ]);
-  option = cond: if cond then "1" else "0";
-in
-stdenv.mkDerivation rec {
+
+assert lib.warnIf (withTools != null) "kissfft: `withTools` is unnecessary and deprecated" true;
```

Comment:
Then just drop it instead. No need to maintain compatibility code for overrides that are likely unused

Reply:
Done.